### DISCUSSION TITLE
Webhooks: Error 500: Cannot convert undefined or null to object

### DISCUSSION THREAD

**Discussion by theneilkirk** [Budibase/budibase#1960] [Help]
Hi

I'm getting started with Budibase and it looks like amazing software!

I am trying to set up a webhook for the first time and got this popup:
![Screenshot 2021-07-09 at 13 15 27](https://user-images.githubusercontent.com/38278363/125076371-bda0c880-e0b7-11eb-804b-e6fe44d8892e.png)

I tried sending a POST request to the URL but I get this error:
`{
    "message": "Cannot convert undefined or null to object",
    "status": 500
}`

Seems like the webhook part is working but failing in some of the config...

Can anyone offer any insights?

I'm running it locally on my mac using docker.

**Comment by mjashanks** [comment]
Hey @theneilkirk 

Thanks for reporting this, it looks like a minor bug. The webhook is expecting a JSON body. 

If you do not need to send any data in the body, you can just skip this step. Otherwise, please submit the body, in JSON, with the correct structure. This will enable the fields in the JSON to be bound to, from other automation steps.
